### DISCUSSION TITLE
Tailwind 4 ❤️

### DISCUSSION THREAD

**Discussion by coolemur** [tailwindlabs/tailwindcss#13181] [Kind words]
It's insane how simple configuration becomes in v4.
I'm always saying "JS is not the place for CSS". And now we have configuration in CSS with variables instead of using config file. ❤️
Much appreciated of what Tailwind team is doing! Can't wait for v4 release.

**Comment by WillianDamasceno** [comment]
I did not tested it yet, does it has IntelliSense in css?

**Reply by philipp-spiess** [reply]
@WillianDamasceno It does! 🙂

**Reply by WillianDamasceno** [reply]
> @WillianDamasceno It does! 🙂 

Wow, that's gonna be amazing once it's launched!
